khal/cli.py

```diff
@@ -93,6 +93,16 @@ def multi_calendar_option(f):
     return d(a(f))
 
 
+def mouse_option(f):
+    o = click.option(
+        '--mouse/--no-mouse',
+        is_flag=True,
+        default=True,
```

Comment:
I believe we need to set the default to `None` and then check explicity in `interactive_cli()` and `interactive()` if `mouse` is set to something other than `None` and only override ctx if that is the case.